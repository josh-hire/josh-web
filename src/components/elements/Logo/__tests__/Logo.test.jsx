import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Logo } from '../index';
jest.mock('next/router', () => ({
  useRouter: () => ({
    query: {
      hl: 'id',
    },
  }),
}));

describe('src/components/elements/Logo', () => {
  test('Logo', () => {
    const { asFragment } = render(<Logo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
