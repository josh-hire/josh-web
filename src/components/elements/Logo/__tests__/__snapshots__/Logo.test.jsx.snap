// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/Logo Logo 1`] = `
<DocumentFragment>
  <img
    alt="Logo PT_tester"
    data-nimg="1"
    decoding="async"
    fit="contain"
    height="46"
    loading="lazy"
    src="/logo-top.svg"
    style="color: transparent;"
    width="190"
  />
</DocumentFragment>
`;
